backend-shared/config/db/utils.go

```diff
@@ -168,3 +169,43 @@ func generateUuid() string {
 func isEmpty(str string) bool {
 	return len(strings.TrimSpace(str)) == 0
 }
+
+// A generic function to validate length of string values in input provided by users.
+// The max length of string is checked using constant variables defined for each type and field in db_field_constants.go
+func ValidateFieldLength(obj interface{}) error {
```

Comment:
Since this function isn't used by other Go packages, it doesn't need to be exported: it can be renamed to `validateFieldLength`.